components/server/src/workspace/workspace-deletion-service.ts

```diff
@@ -69,8 +69,7 @@ export class WorkspaceDeletionService {
         const span = TraceContext.startSpan("garbageCollectWorkspace", ctx);
 
         try {
-            const deleteSnapshots = ws.softDeleted === "user";
```

Comment:
Look at git history, this has been there since the initial gitpod public repo commit.